standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java

```diff
@@ -1454,6 +1454,12 @@ public void createTable(CreateTableRequest request) throws
       request.setProcessorIdentifier(processorIdentifier);
     }
 
+    EnvironmentContext envContext = request.getEnvContext();
+    envContext = Optional.ofNullable(envContext).orElse(new EnvironmentContext());
```

Comment:
`EnvironmentContext envContext = Optional.ofNullable(request.getEnvContext()).orElse(new EnvironmentContext());`
i think that is more readable